tests/dp_computations_test.py

```diff
@@ -544,6 +559,25 @@ def test_gaussian_mechanism_distribution(self, epsilon, delta,
         res = stats.ks_1samp(noised_values, expected_cdf)
         self.assertGreater(res.pvalue, 1e-4)
 
+    def test_gaussian_mechanism_distribution_nd_array(self):
```

Comment:
I wonder whether we can parameterize (probaly not by mechanism but by type of the input)